File: app/src/main/java/org/mifos/selfserviceapp/models/AccountOptionAndBeneficiary.java
package org.mifos.selfserviceapp.models;

import org.mifos.selfserviceapp.models.beneficary.Beneficiary;
import org.mifos.selfserviceapp.models.templates.account.AccountOptionsTemplate;

import java.util.List;

/**
 * Created by dilpreet on 23/6/17.
 */

public class AccountOptionAndBeneficiary {

    private AccountOptionsTemplate accountOptionsTemplate;
    private List<Beneficiary> beneficiaryList;

    public AccountOptionAndBeneficiary(AccountOptionsTemplate accountOptionsTemplate,
                                       List<Beneficiary> beneficiaryList) {
        this.accountOptionsTemplate = accountOptionsTemplate;
        this.beneficiaryList = beneficiaryList;
    }

    public List<Beneficiary> getBeneficiaryList() {
        return beneficiaryList;
    }

    public AccountOptionsTemplate getAccountOptionsTemplate() {

        return accountOptionsTemplate;
    }
}

File: app/src/main/java/org/mifos/mobile/models/AccountOptionAndBeneficiary.kt
package org.mifos.mobile.models

import org.mifos.mobile.models.beneficiary.Beneficiary
import org.mifos.mobile.models.templates.account.AccountOptionsTemplate

/**
 * Created by dilpreet on 23/6/17.
 */

data class AccountOptionAndBeneficiary(
        val accountOptionsTemplate: AccountOptionsTemplate,
        val beneficiaryList: List<Beneficiary>)
